test/python/test_abs.py

```diff
@@ -1,52 +0,0 @@
-# ==============================================================================
-#  Copyright 2018 Intel Corporation
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-# ==============================================================================
-"""nGraph TensorFlow bridge abs operation test
-
-"""
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-import pytest
-
-import tensorflow as tf
-import os
-
-from common import NgraphTest
-
-
-class TestAbsOperations(NgraphTest):
-
-    @pytest.mark.parametrize("test_input", (1.4, -0.5, -1))
-    def test_abs_1d(self, test_input):
-        val = tf.placeholder(tf.float32, shape=(1,))
-        out = tf.abs(val)
-
-        def run_test(sess):
-            return sess.run((out,), feed_dict={val: (test_input,)})
-
-        assert self.with_ngraph(run_test) == self.without_ngraph(run_test)
-
-    def test_abs_2d(self):
-        test_input = ((1.5, -2.5, 0.0, -3.5), (-4.5, -5.5, 6.5, 1.0))
-        val = tf.placeholder(tf.float32, shape=(2, 4))
-        out = tf.abs(val)
-
-        def run_test(sess):
-            return sess.run(out, feed_dict={val: test_input})
-
-        assert (
-            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
```

Comment:
abs, cast and conv2dbackpropinput are the only pytests that would currently run in the deviceless setting. Let us not remove these tests, as it is an example of how to write python tests. Today I had jianying asking me how might one write a pytest in deviceless setting and I pointed her to this. We might need to implement a few ops as pytest (as cpp api might not support it). So lets keep the abs test as an example.

note, the conv2dbackpropinput test exists and uses the ```self.with_ngraph(run_test_ngraph), self.without_ngraph(run_test_tf)``` line, but it is a complex testcase. For a hello-world, this is a good example.

Reply:
ok I'll keep the above mentioned 3 tests